### PR TITLE
feat: Add section for training & certification

### LINKED ISSUE #103: Certifications and Attendances
Currently there looks like no place to enter the certifications we've been given, or conferences/seminars we attended.

I suggest to create two main categories more named "Certifications" and "Attendances".

### PR DESCRIPTION
**Pull request by nij4t**
This pull request introduces a section for training and certificates as well as adds url for awards.

Closes #103 #177 #235 #341

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -248,6 +248,11 @@
             "description": "e.g. 1989-06-12",
             "format": "date"
           },
+          "url": {
```
Comment (evanplaice) [inline schema.json:251]: Adding a `url` to the `awards` section is a separate issue. Remove this.

File: schema.json [hunk 2]
```diff
@@ -259,6 +264,39 @@
...
+            "type": "string",
+            "description": "e.g. http://example.com",
+            "format": "uri"
+          },
+          "issuer": {
+            "type": "string",
+            "description": "e.g. CNCF"
+          },
+          "summary": {
```
Comment (evanplaice) [inline schema.json:293]: A summary shouldn't be necessary for an official certification (see LinkedIn). Remove the `summary` property.

File: schema.json [hunk 3]
```diff
@@ -259,6 +264,39 @@
         }
       }
     },
+    "certificates": {
```
Comment (evanplaice) [inline schema.json:267]: Rename `certificates` -> `certifications`

Comment (stp-ip) [reply schema.json:267]: What's the reasoning here?

Comment (evanplaice) [reply schema.json:267]: Disregard this. I spent some time researching the difference between the two and `certificates` is more broadly applicable.

**Comment by thomasdavis** [comment]
And just adding the original confirmation of certificates to this PR for reference. 
https://github.com/jsonresume/resume-schema/issues/177

**Comment by evanplaice** [comment]
How about. Remove `summary` and add `expires`? The date format needs to be updated to use the new ISO8601 definition but that can be switched over just before merge.

File: schema.json (hunk 1 above)
Comment (nij4t) [reply schema.json]: `url` has been removed from `awards` section.

File: schema.json (hunk 2 above)
Comment (nij4t) [reply schema.json]: I've removed `summary` property.

**Comment by stp-ip** [comment]
```
"certificates": {
      "additionalProperties": false,
      "description": "language section with summary and list of certificates and permits",
      "properties": {
        "certificates": {
          "description": "list of certificates",
          "items": {
            "additionalProperties": false,
            "properties": {
              "code": {
                "description": "code of the certificate",
                "type": "string"
              },
              "description": {
                "description": "short description of the certificate",
                "type": "string"
              },
              "endDate": {
                "description": "end date",
                "format": "date",
                "type": "string"
              },
              "entity": {
                "additionalProperties": false,
                "description": "entity details",
                "properties": {
                  "description": {
                    "description": "entity's primary activity",
                    "type": "string"
                  },
                  "image": {
                    "description": "url or src of an image, photo or logo",
                    "type": "string"
                  },
                  "name": {
                    "type": "string"
                  },
                  "url": {
                    "description": "url of the entity website",
                    "format": "uri",
                    "type": "string"
                  }
                },
                "required": [
                  "name"
                ],
                "type": "object"
              },
              "image": {
                "description": "url or src of a certification badge or another related image",
                "type": "string"
              },
              "name": {
                "description": "name of the certificate",
                "type": "string"
              },
              "perpetual": {
                "description": "validity status being perpetual or not",
                "type": "string"
              },
              "score": {
                "additionalProperties": false,
                "description": "exam score",
                "properties": {
                  "best": {
                    "description": "best possible score",
                    "type": "string"
                  },
                  "label": {
                    "description": "label/type of the score",
                    "type": "string"
                  },
                  "value": {
                    "description": "value of the score",
                    "type": "string"
                  },
                  "worst": {
                    "description": "worst possible score",
                    "type": "string"
                  }
                },
                "required": [
                  "value",
                  "best",
                  "worst"
                ],
                "type": "object"
              },
              "startDate": {
                "description": "start date",
                "format": "date",
                "type": "string"
              },
              "verification": {
                "description": "public certificate verification URL",
                "format": "uri",
                "type": "string"
              }
            },
            "required": [
              "name",
              "entity"
            ],
            "type": "object"
          },
          "type": "array"
        },
        "summary": {
          "description": "continous text summarizing or adding context to the full certificate section",
          "type": "string"
        }
      },
      "type": "object"
    }
```

Maybe good as an inspiration, if nothing else :)

**Comment by DandyDev** [comment]
@thomasdavis @stp-ip do we want to merge this? I'm in favor.

**Comment by thomasdavis** [comment]
@DandyDev agree with `name` consistency instead of `title`, pushing hot fix.

**Comment by thomasdavis** [comment]
Fixed in https://github.com/jsonresume/resume-schema/commit/81d9e75a3d7b599e202e7c3e31c55e634c7f1e39

**Pull request merged**